### DISCUSSION TITLE
How to add curosr position,after `MCstart`

### DISCUSSION THREAD

**Discussion by scc02** [smoka7/multicursors.nvim#22] [Q&A] [answered]
Thank you for the work!
After `MCstart`,How to add curosr position ? In the different column.

**Comment by smoka7** [comment]
That's a good idea, I'll add it.

**Comment by smoka7** [comment, marked as answer]
Now you could do this with `<C-n>`.

**Reply by scc02** [reply]
Is there a way unbind keys? I want unbind jk, This is my workflow:
1. : MCstart
2. `hjkl`move
3. <C-n> Creates a selection for the char under the cursor
4. Into insert mode, editting

**Comment by smoka7** [comment]
Passing this to setup should do it.
```lua
{
 normal_keys ={
    ['j'] = {
             method = false,
    },
}
```
